spec/requests/feedback_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+RSpec.describe 'feedback', type: :request do
+  let(:message_delivery) { instance_double(FeedbackMailer::MessageDelivery) }
+
+  describe 'when clicking the feedback link', type: :request do
+    before { get '/feedback/new' }
+
+    it 'renders feedback/new' do
+      expect(response).to render_template('feedback/new')
+    end
+  end
+
+  describe 'Create feedback', type: :request do
+    let(:feedback_params) do
+      {
+        feedback:
+          {
+            comment: 'A feedback comment',
+            rating: 1,
+            email: 'feedback.user@example.com'
+          }
+      }
+    end
+
+    before do
+      allow(FeedbackMailer).to receive(:notify).and_return(message_delivery)
+      allow(message_delivery).to receive(:deliver_later!)
+      post '/feedback', params: feedback_params
```

Comment:
regarding the environment be good to stub it and check that the request sent included the stubbed value

Reply:
```
expect(a_request(:post, <a_path_or_regexp_path_to_govuknotify>)
  .with(body: <expected_body>).to have_been_made.once
```

Reply:
examples: https://github.com/bblimke/webmock#setting-expectations-in-rspec-with-a_request

Reply:
This is backgrounded so cannot be specs. pushed a commit to test sending the personalisation to #notify instead